source/features/latest-tag-button.tsx

```diff
@@ -18,67 +31,125 @@ const getLatestTag = cache.function(async (): Promise<string | false> => {
 			}) {
 				nodes {
 					name
+					tag: target {
+						oid
+						... on Tag {
+							commit: target {
+								oid
+							}
+						}
+					}
+				}
+			}
+			defaultBranchRef {
+				target {
+					oid
 				}
 			}
 		}
 	`);
 
-	const tags: string[] = repository.refs.nodes.map((tag: {name: string}) => tag.name);
-	if (tags.length === 0) {
-		return false;
+	if (repository.refs.nodes.length === 0) {
+		return {
+			latestTag: false,
+			isUpToDate: false
+		};
 	}
 
-	// If all tags are plain versions, parse them
-	if (tags.every(tag => /^[vr]?\d/.test(tag))) {
-		return tags.sort(compareVersions).pop()!;
+	const tags: Tag[] = repository.refs.nodes.map((node: AnyObject) => ({
+		name: node.name,
+		commit: getTagOid(node)
```

Comment:
I think something like this should be enough:

```suggestion
		commit: node.tag.commit?.oid ?? node.tag.oid
```

Reply:
Cheers! Wasn't familiar with that notation